.github/workflows/week1.yml

```diff
@@ -65,26 +101,52 @@ jobs:
         kubectl get pod 
         
         
+
+  create-cluster-kind-with-minio:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.3.0
+
+      - name: Try the cluster !
+        run: kubectl get pods -A
+      - name: deploy job kind
+        run: |
+          kubectl create -f minio_service/minio_deploy.yml
+          kubectl get pod
+      - name:  deploy app to kind and open port
+        run: |
+          kubectl wait --for=condition=ready pod -l app=minio 
+          kubectl port-forward svc/minio-api 9000:9000  & 
+          kubectl get pod 
+          
         
+   
         
         
         
-  
+    
         
         
+  
+
+        # use '&' for cancel job
         #docker docker login -u ${{secrets.LOGIN_DOCKER_HUB}} -p  ${{secrets.PASSWORD_DOCKER_HUB}}
         #docker pull ${{secrets.LOGIN_DOCKER_HUB}}/${{env.IMAGE_SIMLESERVER}}:latest
         #docker run  --rm -p 8080:8080 --name app-web-Ml-run ${{env.IMAGE_SIMLESERVER}}:latest
-    
-        
+
+
         # to docker build used 'docker build --tag {name:version} [.patch (if it current folder just write '.')]'
         #to deploy docker image use command 'docker run -it --rm -p 80:80 --name {any name } {name:verson}'
-        #to push image to git hub action should use commands: 
+        #to push image to git hub action should use commands:
         #docker login -u ${{secrets.LOGIN_DOCKER_HUB}} -p  ${{secrets.PASSWORD_DOCKER_HUB}}
         #docker tag {image:version}  {folderInDockerHub/image:version}
         #docker push {image:version}
         #to download use:
         #   docker login -u ${{secrets.LOGIN_DOCKER_HUB}} -p  ${{secrets.PASSWORD_DOCKER_HUB}}
         #   docker pull ${{secrets.LOGIN_DOCKER_HUB}}/${{env.IMAGE_SIMLESERVER}}:latest
         #   docker run  --rm -p 8080:8080 --name app-web-Ml-run ${{env.IMAGE_SIMLESERVER}}:latest
-        #   to run with logs should add teg '-it' like this :' docker run -it  --rm -p 8080:8080 --name app-web-Ml-run ${{env.IMAGE_SIMLESERVER}}:latest'
+        #   to run with logs should add teg '-it' like this :' docker run -it  --rm -p 8080:8080 --name app-web-Ml-run ${{env.IMAGE_SIMLESERVER}}:latest'
```

Comment:
remove commented code

Reply:
This code will be help me in further for sort out in my code

Reply:
so keep it in separate branch, in main branch should be only actual code